### PR TITLE
Supernode images

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Added support to link images to entries.

Example of use http://node2.e-mesh.net:4000/

### CODE REVIEW

**Comment by benhylau** [comment]
Cool, I guess this should be an optional field in https://github.com/tomeshnet/node-list/blob/master/schema/v0.6/schema.json

**Comment by darkdrgn2k** [comment]
note really sure how to define an array in json, can you take a look

also wondering if we would like additional fields, like maybe caption? or maybe "type of image" or something.

**Comment by makeworld-the-better-one** [comment]
In chat we discussed how these larger images sometimes take time to load, which feels strange UX-wise because they are loading in a small space on the map. One simple solution besides generating thumbnails would be to convert the images to progressive JPEGs, so that the whole image could load at once, instead of from top to bottom. Progressive JPEGs are actually smaller than regular ones, so that would be a bonus.

The animation below mentions interlacing, which is for PNGs, but the idea is the same.
BtI3d

File: schema/v0.6/schema.json [hunk 1]
```diff
@@ -104,6 +104,16 @@
                         ],
                         "pattern": "^.*$"
                     },
+                    "images": {
+                        "$id": "#root/nodeList/items/images", 
+                        "title": "Images", 
+                        "type": "object",
+                        "default": "",
```
Comment (makeworld-the-better-one) [inline schema/v0.6/schema.json:111]:
```suggestion
                        "type": "array",
                        "default": [],
```

File: schema/v0.6/schema.json [hunk 2]
```diff
@@ -104,6 +104,16 @@
...
+                        "examples": [
+                            "sn1a2-hw.jpg"
```
Comment (makeworld-the-better-one) [inline schema/v0.6/schema.json:113]:
```suggestion
                            ["sn1a2-hw.jpg"]
```

**Comment by makeworld-the-better-one** [comment]
This can be closed in favour of #32.

**Pull request closed without merging**